src/com/sap/piper/DefaultValueCache.groovy

```diff
@@ -5,26 +5,36 @@ import com.cloudbees.groovy.cps.NonCPS
 class DefaultValueCache implements Serializable {
     private static DefaultValueCache instance
 
+    // Contains defaults values provided by this library itself
     private Map defaultValues
 
-    private DefaultValueCache(Map defaultValues){
+    // intended for describing e.g. the system landscape on customer side
+    private Map customDefaultValues
+
+    private DefaultValueCache(Map customeDefaultValues, Map customDefaultValues){
```

Comment:
Any reason why we need to change this? We could only store an already merged version in the `DefaultValueCache`.

I don't have a use case in mind where we need to distinguish later on if we deal with default-default or default-custom ...

Thus I would suggest to keep the cache as is

Reply:
This is also possible, of course. i'm not sure if there are use cases where we would like to see (only) the defaults delivered with the piper-lib. If nobody has such a use case we can also merge before creating the cache instance.

Reply:
+1 for merging before creating the cache instance